R/series__series.R

```diff
@@ -679,6 +705,26 @@ Series_max = "use_extendr_wrapper"
 #' pl$Series(c(1:2,3,Inf,4,-Inf,5))$min() # Inf-Inf is NaN
 Series_min = "use_extendr_wrapper"
 
+#' @title Var
+#' @description Aggregate the columns of this Series to their variance values.
+#' @keywords Series
+#' @param ddof integer Delta Degrees of Freedom: the divisor used in the calculation is N - ddof, where N represents the number of elements. By default ddof is 1.
+#' @return A new `Series` object with applied aggregation.
+#' @examples pl$Series(1:10)$var()
+Series_var = function(ddof = 1) {
+  unwrap(.pr$Series$var(self, ddof))
```

Comment:
we can add a context in unwrap like this
`unwrap(.pr$Series$var(self, ddof), "in $var()")`

my thinking behind adding context to errors is that it can otherwise be hard to locate the source of an error inside a long chain of several methods. If new users get confused, they might give up on polars prematurely. I have not consistently added context to all methods, but I think it is helpful.

Reply:
e7721fb018caaa05968d25bbe90166c2217437c2 adds context to these and several others where it was originally missing.